lancero_source.go

```diff
@@ -121,6 +124,9 @@ type LanceroSourceConfig struct {
 	CardDelay         []int
 	ActiveCards       []int
 	ShouldAutoRestart bool
+	FirstRow          int // Channel number of the 1st row (default 1)
+	ChanSepCards      int // Channel separation between cards (or 0 to indicate number sequentially)
+	ChanSepColumns    int // Channel separation between columns (or 0 to indicate number sequentially)
```

Comment:
If I want your example of `12034` for card 1, col 2, row 34, I put in FirstRow=1, ChanSepCards = 10000, ChanSepColumns=1000?

Reply:
Right. Whereas if you were handling only 32 rows at a time, and this is row 2 in the second group. In that case, you'd say FirstRow=33, but the other two would be like you said.